DOCUMENTATION.md

```diff
@@ -980,6 +990,7 @@ The basis of this script is to provide a simple mechanism for identifying object
 
 #### Grab Mechanics
   * **Grab Attach Type:** This determines how the grabbed item will be attached to the controller when it is grabbed.
+   * `Climbable` non-rigid body interactable object used to allow player climbing
```

Comment:
This should go at the bottom of the list as it's bottom in the actual inspector list